lib/pingCodeOwnerAfter24hrs.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Can this functionality be moved to the periodic check module?

Reply:
Sir, It can be moved, I can do that if you want. Initially, I also thought of doing the same but then I think its a separate functionality just like all the others in the lib folder, That's why I made a different File for that. I would like to know your suggestions on this one sir. If you say then I can transfer the same to periodic check module.